### PR TITLE
tweak(keyviz): add enable tips

### PR DESCRIPTION
**Pull request by shhdgit**
Add keyviz enable tips for better user understanding.
image

### CODE REVIEW

File: ui/lib/apps/KeyViz/translations/en.yaml [hunk 1]
```diff
@@ -26,6 +26,7 @@ keyviz:
     close_keyviz: Disable Key Visualizer Feature
     close_keyviz_warning: Are you sure want to disable this feature? Current visual reports will be cleared.
     switch: Enable
+    switch_tooltip: Turning on Key Visualizer will introduces a small overhead.
```
Comment (shhdgit) [inline ui/lib/apps/KeyViz/translations/en.yaml:29]: How much overhead is there?

Comment (breeswish) [reply ui/lib/apps/KeyViz/translations/en.yaml:29]: No idea..

```suggestion
    switch_tooltip: Whether Key Visualizer feature is enabled. When enabled, there will be small overhead.
```

File: ui/lib/apps/KeyViz/translations/zh.yaml [hunk 2]
```diff
@@ -26,6 +26,7 @@ keyviz:
     close_keyviz: 关闭流量可视化功能
     close_keyviz_warning: 确认要关闭该功能吗？关闭后现有历史记录也将被清空！
     switch: 总开关
+    switch_tooltip: 开启流量可视化将会增加少量资源开销
```
Comment (breeswish) [inline ui/lib/apps/KeyViz/translations/zh.yaml:29]:
```suggestion
    switch_tooltip: 是否开启流量可视化，流量可视化启用时将会增加少量资源开销
```

**Review by breeswish — APPROVED** [review]

**Review by breeswish — APPROVED** [review]

**Comment by breeswish** [comment]
I think we can further add more tips for similar panels, like the Statement panel :)

**Comment by shhdgit** [comment]
Yeah, sure.

**Pull request merged**